app/screens/search/index.js

```diff
@@ -22,6 +22,18 @@ function makeMapStateToProps() {
 
     return (state) => {
         const postIds = preparePostIds(state, state.entities.search.results);
+
+        const channels = {};
+        postIds.forEach((postID) => {
```

Comment:
also not a fan, this will cause a huge amount of re-renders just because every time this is going to be a new object.

can you make this a selector factory and then use the selector and instead of having the whole object for every key just hold if it was deleted or not?

Reply:
Sure can.